en/Releases/index.md

```diff
@@ -17,12 +17,12 @@ lastUpdated: 2018-05-15
 * Added "asset search bar" to the dashboard of enebular.
 * Logs from enebular edge agent can now be displayed on enebular.
 
-## Fixed
+### Fixed
 
 * All the screenshots in our English manual were captured from Japanese version of the enebular dashboard, and we have replaced them all with the ones from the English version.
```

Comment:
* English documentation of Enebular Dashboard now contains screen shots in English (Previously displaying screen shots in Japanese).